injector/network_disruption.go

```diff
@@ -198,6 +246,8 @@ func (i networkDisruptionInjector) Clean() error {
 //           |- (4:) <-- second operation
 //             ...
 func (i *networkDisruptionInjector) applyOperations() error {
+	i.tcFilterPriority = 49150
```

Comment:
Magic number? Could you define this constant at the top of the file in that case? Thanks!

Reply:
This is in the range of the numbers assigned by tc automatically! I'll add a comment on that